static/app/utils/handleXhrErrorResponse.tsx

```diff
@@ -1,41 +1,24 @@
 import * as Sentry from '@sentry/react';
 
-import {ResponseMeta} from 'sentry/api';
+import RequestError from './requestError/requestError';
 
-export default function handleXhrErrorResponse(message: string) {
-  return (resp: ResponseMeta) => {
-    if (!resp) {
-      return;
-    }
-    if (!resp.responseJSON) {
-      return;
-    }
+export default function handleXhrErrorResponse(resp: RequestError) {
+  if (!resp) {
+    return;
+  }
+  if (!resp.responseJSON) {
+    return;
+  }
 
-    const {responseJSON} = resp;
-
-    // If this is a string then just capture it as error
-    if (typeof responseJSON.detail === 'string') {
-      Sentry.withScope(scope => {
-        scope.setExtra('status', resp.status);
-        scope.setExtra('detail', responseJSON.detail);
-        Sentry.captureException(new Error(message));
-      });
-      return;
-    }
+  const {responseJSON} = resp;
 
+  // If this is a string then just capture it as error
+  if (typeof responseJSON.detail !== 'string') {
     // Ignore sudo-required errors
     if (responseJSON.detail && responseJSON.detail.code === 'sudo-required') {
```

Comment:
I originally removed this fn completely, but thought it might be worth to keep for this case.